xunit-test/NearCacheTest/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ target_link_libraries(NearCacheMultiClientTest
 add_test (start_server ${PYTHON_EXECUTABLE} ${CMAKE_ROOT_SOURCE_DIR}/test/bin/server_ctl.py start ${JAVA_RUNTIME} ${HOTROD_JBOSS_HOME} standalone.xml)
 add_test (probe_port ${PYTHON_EXECUTABLE} ${CMAKE_ROOT_SOURCE_DIR}/test/bin/probe_port.py localhost 11222 60)
 
-add_test(xunit_nearCacheMultiClientTest NearCacheMultiClientTest --gtest_output=xml:${PROJECT_BINARY_DIR}/test-output/)
+add_test(xunit_nearCacheTest xunit_nearCacheTest --gtest_output=xml:${PROJECT_BINARY_DIR}/test-output/)
 
 add_test (stop_server ${PYTHON_EXECUTABLE} ${CMAKE_ROOT_SOURCE_DIR}/test/bin/server_ctl.py stop)
 add_test (probe_port_stop ${PYTHON_EXECUTABLE} ${CMAKE_ROOT_SOURCE_DIR}/test/bin/probe_port.py localhost 11222 60 down)
```

Comment:
`NearCacheEvictionTest` is not being executed

Reply:
@alanfx the tests fail randomically, I'm looking into.